### DISCUSSION TITLE
Lung blood flow rates

### DISCUSSION THREAD

**Discussion by Supercn01** [Open-Systems-Pharmacology/Forum#1691] [Models]
Hi everyone!
   I want change the blood flow rates of lung in my model but I cannot modify this parameters. Would anyone tell me how do? 
I will appreciated!

Thankyou!

**Comment by tobiasK2001** [comment]
Hi,
actually the blood flow rate in lung is the sum of the organ specific blood flows of the organs which are supplied with fresh blood from the lungs. You can see the formula and the organs by mouse over the lung blood flow. 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/1d310fd9-e4ff-4b53-bfa8-483a70f3e0df)
So if you increase e.g. the bone blood flow, this is automatically reflected in the lung blood flow. Maybe the best way would be to increase all involved organs by the same factor to get the lung blood flow you want. 

Hope that might help you.

Best, Tobias

**Reply by Supercn01** [reply]
Thank you very much for your help, tobias. I'll give it a try
